android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java

```diff
@@ -235,6 +237,25 @@ public void onError(@NonNull RegistrationException error,
         };
     }
 
+    private UnregistrationListener unregistrationListener() {   
+        return new UnregistrationListener() {
+            @Override
+            public void onUnregistered(String accessToken, String fcmToken) {
+                if (BuildConfig.DEBUG) {
+                    Log.d(TAG, "Successfully unregistered FCM");
+                }
+            }
+
+            @Override
+            public void onError(RegistrationException error, String accessToken, String fcmToken) {
+                Log.e(TAG, String.format("Unregistration Error: %d, %s", error.getErrorCode(), error.getMessage()));
+                WritableMap params = Arguments.createMap();
+                params.putString(Constants.ERROR, error.getMessage());
+                eventManager.sendEvent(EVENT_DEVICE_NOT_READY, params);
```

Comment:
EVENT_DEVICE_NOT_READY is not the right event for this error. Since, you removed the JS feedback for successful unregistration, I think we should also remove the feedback for error.
However, if you think that this feedback is needed, I would recommend to add a proper event name and mapping, including README docs.

Reply:
I removed the following part:
- WritableMap params = Arguments.createMap();
-  params.putString(Constants.ERROR, error.getMessage());
- eventManager.sendEvent(EVENT_DEVICE_NOT_READY, params);